BraintreePSP/BraintreePaymentNonceProvider.swift

```diff
@@ -136,6 +136,9 @@ final class BraintreePaymentNonceProvider: PaymentNonceProvider {
                 triggerAddCardFlow(currencyCode: quote.price.currencyCode)
             case .success(let threeDSecureNonce):
                 self.callbackResult?(.completed(value: .nonce(nonce: Nonce(nonce: threeDSecureNonce))))
+            case .canceledByUser:
```

Comment:
Why we need this value? Wne it will be used? As you described below, it shouldn't be possible.

Reply:
It's possible in NewCheckoutPresenter, not PSP object. But after all it's good point, there is no point to keep that case.